### DISCUSSION TITLE
Flex behaves differently on tailwind and vanilla CSS

### DISCUSSION THREAD

**Discussion by afkcodes** [tailwindlabs/tailwindcss#6870] [Help] [answered]
I am creating a basic Flex using tailwind and using Vanilla CSS
The tailwinds implementation seems a bit different

Here is the Vanilla Implementation
[Vanilla implementation](https://imgur.com/pI9U7rA)

Here is the Tailwind Implementation
[Tailwind Implementation ](https://imgur.com/1YMZ5Q8)

in vanilla the items inside the flex container is overflowing out of the parent if i add more number of items;
where as in tailwind it squeezes the items to make it fit into the parent, i  wanted to know how this is happening and how can i get the default vanilla behaviour. i am doing it wrong or tailwind is doing some magic.

**Comment by adamwathan** [comment, marked as answer]
Hey! I don't think this is true, the `flex` class sets `display: flex` that's it. Here's a demo of the exact same code using the `flex` class and an inline style that sets `display: flex` and you can see the behavior is the same:

https://play.tailwindcss.com/T8xA3OBo6W

Happy to look at a demo you create if you're seeing otherwise, unfortunately the Imgur screenshot isn't really enough to do anything with.

**Reply by afkcodes** [reply]
Thanks @adamwathan i will surely have a look at this im probably doing something wrong then thanks for the update.
